elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -106,10 +116,18 @@ public void commit() {
     @Override
     public <T> T loadObject(Class<T> loadClass, Serializable id) {
         try {
-            @SuppressWarnings("unchecked")
-            T record = (T) session.load(loadClass, id);
-            Hibernate.initialize(record);
-            return record;
+            if (isJoinQuery()) {
```

Comment:
I thought the plan was to have some header or request param to enable/disable the join?